core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -242,7 +249,22 @@ class HttpGoogleServicesDAO(
     val traceId = TraceId(UUID.randomUUID())
 
     for {
-      _ <- traceWithParent("insertBucket", parentSpan)(_ => googleStorageService.insertBucket(GoogleProject(googleProject.value), GcsBucketName(bucketName), None, labels, Option(traceId), true, Option(GcsBucketName(getStorageLogsBucketName(googleProject)))).compile.drain.unsafeToFuture()) //ACL = None because bucket IAM will be set separately in updateBucketIam
+      _ <- traceWithParent("insertBucket", parentSpan)(_ => googleStorageService.insertBucket(
+        googleProject = GoogleProject(googleProject.value),
+        bucketName = GcsBucketName(bucketName),
+        acl = None,
+        labels = labels,
+        traceId = Option(traceId),
+        bucketPolicyOnlyEnabled = true,
+        logBucket = Option(GcsBucketName(getStorageLogsBucketName(googleProject))),
+        location = bucketLocation
+      ).compile.drain.unsafeToFuture().recoverWith{
+        case e: StorageException if e.getCode == 400 =>
+          val message = s"Workspace creation failed. Error trying to create bucket `$bucketName` in Google project " +
+            s"`${googleProject.value}` in region `${bucketLocation.getOrElse("US (default)")}`."
+          val errorReport = ErrorReport(statusCode = StatusCodes.BadRequest, message)
+          throw new RawlsExceptionWithErrorReport(errorReport)
```

Comment:
```suggestion
          throw new RawlsExceptionWithErrorReport(errorReport, e)
```

Reply:
the constructor for `RawlsExceptionWithErrorReport` takes only one argument (see https://github.com/broadinstitute/rawls/blob/d72a788693bf9892210f49871bdb7dfa16b148a0/model/src/main/scala/org/broadinstitute/dsde/rawls/RawlsException.scala#L7)

Reply:
We specifically chose this class because it doesn't return the whole error, and we wanted to avoid showing the entire stack trace to the user.